### PR TITLE
Memory-pair game

### PR DESCRIPTION
**Pull request by ivarshavets**
### Memory-pair game

[Demo](https://ivarshavets.github.io/kottans-frontend/memory-pair-game/index.html) |
[Code base](https://github.com/ivarshavets/kottans-frontend/tree/master/memory-pair-game)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by A-Ostrovnyy** [comment]
Demo not working.

**Comment by ivarshavets** [comment]
@A-Ostrovnyy Sorry, updated the link

File: submissions/ivarshavets/memory-pair-game/js/app.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	const DELAY = 600
+	const cardsHolder = document.querySelector('.cards-holder')
+	const cards = [...data, ...data]
+	const cardActiveClass = 'card--active'
+
+	const renderCards = (cards) => {
+		cards.sort(() => 0.5 - Math.random())
+
+		cards.forEach(card => {
+			const cardEl = document.createElement('div')
```
Comment (A-Ostrovnyy) [inline submissions/ivarshavets/memory-pair-game/js/app.js]: Use template literals to concatenate strings.

Comment (ivarshavets) [reply submissions/ivarshavets/memory-pair-game/js/app.js]: Changed.

**Comment by A-Ostrovnyy** [comment]
Nice work!

**Pull request merged**